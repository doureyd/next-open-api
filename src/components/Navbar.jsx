import { Flex } from '@chakra-ui/react'
import NavLink from '@/components/NavLink'

const Navbar = () => (
  <Flex width="100%" as="nav" paddingY="6">
    <NavLink href="/" passHref>
      Home
    </NavLink>
    <NavLink href="/podcasts" passHref>
      Podcasts
    </NavLink>
    <NavLink href="/book-lists" passHref>
      Book Lists
    </NavLink>
  </Flex>
)

export default Navbar
