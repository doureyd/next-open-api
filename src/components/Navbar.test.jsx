import React from 'react'
import Navbar from '@/components/Navbar'
import { render } from 'test-utils'

jest.mock('next/router', () => ({
  useRouter: () => ({
    asPath: '/',
  }),
}))

describe('Navbar', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Navbar />)

    expect(asFragment()).toMatchSnapshot()
  })
})
