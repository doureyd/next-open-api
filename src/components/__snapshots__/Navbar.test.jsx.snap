// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="css-1xqcw9r"
  >
    <a
      class="chakra-link css-1015fv"
      href="/"
    >
      Home
    </a>
    <a
      class="chakra-link css-12y3vhu"
      href="/podcasts"
    >
      Podcasts
    </a>
    <a
      class="chakra-link css-12y3vhu"
      href="/book-lists"
    >
      Book Lists
    </a>
  </nav>
</DocumentFragment>
`;
